schema/schema.hh

```diff
@@ -934,6 +934,10 @@ public:
      * (and `ALTER ADD` if the column has been re-added) to the description.
      */
     virtual std::ostream& describe(replica::database& db, std::ostream& os, bool with_internals) const override;
+    // Genereate `ALTER TABLE ... WITH` statement that include all table's options
+    sstring alter_table_with_options(replica::database& db) const;
```

Comment:
It will _only_ include the `WITH` part. I think that this is a key property of the function that needs to be noted in the comment.